Discovery attempt against the device with missing SysOID crashes process

Attempted to discovery some custom appliance and got an error.
After some additional troubleshooting, it seems that the cause is missing SysOID value:

```
ktranslate [Debug] KTranslate pdu: {Name:.1.3.6.1.2.1.1.1.0 Type:OctetString Value:[65 99 99 101 115 115 32 71 97 116 101 119 97 121 32 83 101 99 117 114 105 116 121 32 65 112 112 108 105 97 110 99 101]}
2021-09-23T23:48:59.193 ktranslate [Debug] KTranslate pdu: {Name:.1.3.6.1.2.1.1.2.0 Type:NoSuchObject Value:<nil>}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x98 pc=0xbdf52a]

goroutine 49 [running]:
github.com/kentik/ktranslate/pkg/inputs/snmp.doubleCheckHost(0xc00157256c, 0x4, 0x4, 0x2825d28, 0x0, 0x0, 0x2825d28, 0x0, 0x0, 0x2825d28, ...)
        /home/runner/work/ktranslate/ktranslate/pkg/inputs/snmp/disco.go:224 +0x86a
created by github.com/kentik/ktranslate/pkg/inputs/snmp.Discover
        /home/runner/work/ktranslate/ktranslate/pkg/inputs/snmp/disco.go:107 +0xdd8
```

Not sure how this should be handled, but it should not crash the process.

--- a/pkg/formats/nrm/nrm.go
+++ b/pkg/formats/nrm/nrm.go
@@ -393,6 +393,7 @@ func (f *NRMFormat) fromKSynth(in *kt.JCHF) []NRMetric {
 				"max":   uint64(lost),
 			},
 			Attributes: attr,
+			Interval:   60 * 1000000,
 		})
 	}
 

--- a/pkg/inputs/snmp/disco.go
+++ b/pkg/inputs/snmp/disco.go
@@ -221,7 +221,7 @@ func doubleCheckHost(result scan.Result, timeout time.Duration, ctl chan bool, m
 			device.DeviceOids = mibs
 		}
 		// Use the profile's provider if it is set.
-		if mibProfile.Provider != "" {
+		if mibProfile != nil && mibProfile.Provider != "" {
 			device.Provider = mibProfile.Provider
 		} else {
 			device.Provider = provider

